lib/fluent/plugin/kafka_plugin_util.rb

```diff
@@ -42,6 +46,16 @@ def read_ssl_file(path)
         end
       end
 
+      END_CERTIFICATE = "\n-----END CERTIFICATE-----\n"
+
+      def read_ssl_ca_certs
+        return nil if @ssl_ca_cert.nil?
+        valid_paths = @ssl_ca_cert.reject { |path| invalid_path?(path) }
+        valid_paths.flat_map do |path|
+          File.read(path).split(END_CERTIFICATE).map { |c| c += END_CERTIFICATE }
```

Comment:
Since it recognizes only `-----END CERTIFICATE-----`, it can't safely ignore other labels such as `RSA PRIVATE KEY`.
How about using `String#scan` like this:

```ruby
pem = File.read(path)
pem.scan(/-----BEGIN CERTIFICATE-----\n[\s\S]+?-----END CERTIFICATE-----\n/).to_a
```